### PR TITLE
added sample install

### PR DESCRIPTION
**Pull request by kathpanderson**
Added sample install code that includes setting basic prereqs and executing install.

### CODE REVIEW

File: deployment/install/linux.rst [hunk 1]
```diff
@@ -30,3 +30,16 @@ Common Additional Options:
...
+
+Sample code for a complete install with setup included::
+
+	sudo apt-get update
+	sudo apt-get install git
+	mkdir digitalrebar
+	git clone https://github.com/rackn/digitalrebar-deploy digitalrebar/deploy
+	ln -s digitalrebar/ digitalrebar/deploy/compose/digitalrebar
+	cd digitalrebar/deploy
+	ip -4 addr
```
Comment (galthaus) [inline deployment/install/linux.rst]: Add a this:
# Find the ip address of your primary interface from the ip command an use it as the CIDR below.

**Comment by kathpanderson** [comment]
That's already explained in the step by step instructions above the code... I can still add it in again if you think that it isn't covered well enough/if you want it added in again, though. Probably like:

```
echo "find CIDR"
ip -4 addr
echo "save CIDR xx.x.x.xx/## from above"
export IPA=[CIDR] 
```

**Comment by galthaus** [comment]
I'm worried about people mousing that list and failing because CIDR isn't filled in.

**Comment by kathpanderson** [comment]
Ah. I see what you mean. I will add a note at the top then.

**Comment by kathpanderson** [comment]
Fixed in the new commit.

**Pull request merged**